spec/factories/services.rb

```diff
@@ -10,19 +10,22 @@
     feedback_mail_sent { false }
     service_specification
     civil_servant_agreed { true }
-    civil_servant_agreed_on { '2018-08-05' }
     organization_agreed { true }
-    organization_agreed_on { '2018-07-05' }
     association :civil_servant, :full
 
+    trait :decided_dates_set do
+      civil_servant_decided_at { 2.weeks.ago }
+      organization_decided_at { 3.weeks.ago }
+    end
```

Comment:
I don't really get why this trait is necessary. Seems like it's only there because of the model spec. If it is because of that, I'd prefer to put it in the built model there directly

Reply:
True, I'll change it